bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -68,6 +68,11 @@ spec:
           requests:
             cpu: 100m
             memory: 300Mi
+        env:
```

Comment:
@JacobTanenbaum don't need the env vars for the northd pod here.

Reply:
done

Reply:
@JacobTanenbaum looks like they are still there?